package.json

```diff
@@ -10,6 +10,8 @@
     "lint:fix": "next lint  --dir . --fix",
     "test:unit": "jest",
     "test:accessibility": "pa11y-ci -c ./tests/accessibility/.pa11y-ci.js",
+    "test:local-ui": "npx playwright test --project component -c tests/test-team/config/local.config.ts",
```

Comment:
Should these be their own project/package.json - given that you can see some application errors during tests?

Reply:
Updated to separate out package.json and removed npx bit